cmd/boostx/sector.go

```diff
@@ -92,3 +94,71 @@ var sectorUnsealCmd = &cli.Command{
 		return nil
 	},
 }
+
+var isUnsealedCmd = &cli.Command{
+	Name:        "is-unsealed",
+	Usage:       "boostx sector is-unsealed <Sector ID>",
+	Description: "Check if a particular sector is unsealed using the miner API. This is a definitive method to check for unsealed copies",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:     "api-fullnode",
+			Usage:    "the endpoint for the full node API",
+			Required: true,
+		},
+		&cli.StringFlag{
+			Name:     "api-storage",
+			Usage:    "the endpoint for the storage node API",
+			Required: true,
+		},
+	},
+	Action: func(cctx *cli.Context) error {
+
+		if !cctx.Args().Present() {
+			return fmt.Errorf("must pass sector ID as first parameter")
+		}
+
+		sectorIDInt, err := strconv.Atoi(cctx.Args().First())
+		if err != nil {
+			return fmt.Errorf("parsing sector ID %s: %w", cctx.Args().First(), err)
+		}
+		sectorID := abi.SectorNumber(sectorIDInt)
+
+		ctx := lcli.ReqContext(cctx)
+
+		// Connect to the full node API
+		fnApiInfo := cctx.String("api-fullnode")
+		fullnodeApi, ncloser, err := lib.GetFullNodeApi(ctx, fnApiInfo, log)
+		if err != nil {
+			return fmt.Errorf("getting full node API: %w", err)
+		}
+		defer ncloser()
+
+		err = lib.CheckFullNodeApiVersion(ctx, fullnodeApi)
+		if err != nil {
+			return err
+		}
+
```

Comment:
For simplicity I'd suggest we don't check the api version - probably this particular API is not going to change, and this way the SP doesn't have to worry about getting the version of the API in boostx to exactly match the API version in lotus-miner

Reply:
Shall I remove it from https://github.com/filecoin-project/boost/blob/a0509fa9dade5ba84340f376bac79399b2a947a8/cmd/boostx/sector.go#L59 as well? The use case as same.

Reply:
Sure sounds good 👍